### PR TITLE
Handle Entities

### PR DESCRIPTION
**Pull request by JaCkIsO**

### CODE REVIEW

File: src/main/java/domain/Admin.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import javax.persistence.Id;
+import java.util.List;
+
+
+public class Admin {
+
+    @Id
+    long id;
+
+    private List<Review> reviews;
```
Comment (JaCkIsO) [inline src/main/java/domain/Admin.java:13]: Why do we need reviews here?

File: src/main/java/domain/Order.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    @JoinColumn(name="buyer_id")
+    private Buyer buyer;
+
+    @ManyToOne
+    @JoinColumn(name="seller_id")
+    private Seller seller;
+
+    @ManyToOne
+    @JoinColumn(name="order_id")
+    private ShoppingCart shoppingCart;
```
Comment (JaCkIsO) [inline src/main/java/domain/Order.java:25]: This has to be unidirectional from shopping cart side

File: src/main/java/domain/Admin.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package domain;
```
[the remark this answers is not in the document]
Comment (JaCkIsO) [reply src/main/java/domain/Admin.java:1]: Could we also rename com.example to edu.miu

File: src/main/java/domain/Admin.java (hunk 1 above)
Comment (beeaklilu) [reply src/main/java/domain/Admin.java]: Removed

**Pull request merged**